annotations/firefox_desktop_background_update/README.md

```diff
@@ -0,0 +1,5 @@
+As of this writing, the Firefox background updater is Windows-only.
```

Comment:
It'd be nice to put in a sentence like what :nalexander said about how this is operational data about the background update task itself (maybe link to [its docs](https://firefox-source-docs.mozilla.org/toolkit/mozapps/update/docs/BackgroundUpdates.html)) and is not designed to answer questions about user update behaviour.